topi/python/topi/bifrost/conv2d.py

```diff
@@ -142,13 +142,14 @@ def _schedule_spatial_pack(cfg, s, output, conv, data_vec, kernel_vec):
         s[data_vec].unroll(vw)
 
     if isinstance(kernel_vec.op, tvm.te.ComputeOp) and kernel_vec.name == 'kernel_vec':
+        co, ci, kh, kw, vc = s[kernel_vec].op.axis
```

Comment:
Looks like this part share the logic with arm_cpu and mali. We can move this into [conv2d_nchw_spatial_pack compute](https://github.com/apache/incubator-tvm/blob/master/topi/python/topi/arm_cpu/conv2d_spatial_pack.py#L27). An example is in x86 conv2d: https://github.com/apache/incubator-tvm/blob/master/topi/python/topi/x86/conv2d.py#L188-L201

With this method, we can fix mali/arm_cpu/bifrost in one shot.

Reply:
Actually I'm not quite sure whether we can directly do this kind of placeholder replacement inside schedule. https://github.com/apache/incubator-tvm/pull/5511 for mali might also has issue. Put placeholder replacement in compute would be more convenient.

Reply:
Hi @kevinthesun 
Thanks for the review, I move the placeholder replacement to compute.